crates/next-dev-tests/tests/integration/next/font-google/basic/pages/index.js

```diff
@@ -25,27 +28,50 @@ function runTests() {
   });
 
   it("includes a rule styling the exported className", () => {
-    const selector = `.${CSS.escape(interNoArgs.className)}`;
-
-    let matchingRule;
-    for (const stylesheet of document.querySelectorAll(
-      "link[rel=stylesheet]"
-    )) {
-      const sheet = stylesheet.sheet;
-      if (sheet == null) {
-        continue;
-      }
-
-      for (const rule of sheet.cssRules) {
-        if (rule.selectorText === selector) {
-          matchingRule = rule;
-          break;
-        }
-      }
-    }
-
+    const matchingRule = getRuleMatchingClassName(interNoArgs.className);
     expect(matchingRule).toBeTruthy();
     expect(matchingRule.style.fontFamily).toEqual("__Inter_34ab8b4d");
     expect(matchingRule.style.fontStyle).toEqual("normal");
   });
+
+  it("supports declaring a css custom property (css variable)", () => {
+    expect(interWithVariableName).toEqual({
+      className:
+        "className◽[project-with-next]/crates/next-dev-tests/tests/integration/next/font-google/basic/[embedded_modules]/@vercel/turbopack-next/internal/font/google/inter_c6e282f1.module.css",
+      style: {
+        fontFamily: "'__Inter_c6e282f1'",
+        fontStyle: "normal",
+      },
+      variable:
+        "variable◽[project-with-next]/crates/next-dev-tests/tests/integration/next/font-google/basic/[embedded_modules]/@vercel/turbopack-next/internal/font/google/inter_c6e282f1.module.css",
+    });
+
+    const matchingRule = getRuleMatchingClassName(
+      interWithVariableName.variable
+    );
+    expect(matchingRule.styleMap.get("--my-font").toString().trim()).toBe(
+      '"__Inter_c6e282f1"'
+    );
+  });
+}
+
+function getRuleMatchingClassName(className) {
+  const selector = `.${CSS.escape(className)}`;
+
+  let matchingRule;
+  for (const stylesheet of document.querySelectorAll("link[rel=stylesheet]")) {
+    const sheet = stylesheet.sheet;
+    if (sheet == null) {
+      continue;
```

Comment:
There is a race condition with the loading of stylesheets here. This makes the tests flaky.

Maybe add a `await new Promise(resolve => stylesheet.onload = resolve);` or similar here?